test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/V3DetailControlModelTests.cs

```diff
@@ -131,6 +131,19 @@ public void VulnerabilityCountWhenMetadataHasVulnerability_Calculated()
             Assert.Equal(_testInstance.PackageVulnerabilityCount, _testData.TestData.Vulnerabilities.Count());
         }
 
+        [Fact]
+        public void PackageVulnerabilities_WhenMetadataHasVulnerability_IsOrderedBySeverityDescending()
+        {
+            var sortedTestVulnerabilities =
+                _testData.TestData.Vulnerabilities
+                .OrderByDescending(v => v.Severity)
+                .Select(v => new Action<PackageVulnerabilityMetadataContextInfo>(
```

Comment:
I mentioned this in a previous review of this PR, but typically we break apart Arrange/Act/Assert ("AAA") into separate sections (see coding guideline https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#unit-test-structure )

Wrapping the delegate here, relying on xunit's `Assert.Collection` to invoke those delegates, while technically possible, isn't very easy to read or follow what this test is doing.
Consider switching to the "AAA" pattern, and instead using a more declarative assert like `Assert.Equal(IEnumerable<Severity>, IEnumerable<Severity>)`.

Reply:
Thanks for the feedback! The "act" portion of this is part of the setup and is in the Test class's constructor where it calls `_testInstance.SetCurrentPackageAsync` to set it to the `_testViewModel` and internally that ends up populating the Vulnerabilities list which I'm asserting is in the proper order. I'm going to leave the test instance as it was for this class already so the act part will be common for all the tests as it was written before. Changing that would require refactoring the test object and its common setup that runs through the product code (act) and the properties get their appropriate values, which are what a lot of these tests are validating.

Based on your suggestion, I split the code up and use Assert.Equal instead of the Assert.Collection which requires the use of an element inspector. Thanks!